cypress/e2e/ModelizerPage/SwitchView/add.component.feature

```diff
@@ -0,0 +1,123 @@
+    And  I expect "[data-cy=\"file-label-leto-modelizer.config.json\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"inactive-tab\"]" is "leto-modelizer.config.json"
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "default.plugin-test.js"
+    And  I expect active file content to contain "\"type\": \"truck\""
+
+  Scenario: Update plugin file's content with a new object (Text view) should display the corresponding plugin component (Model view)
+    When I click on "[data-cy=\"component-definition-truck\"]"
+    And  I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Text"
+    And  I expect "[data-cy=\"modelizer-text-view\"]" exists
+    And  I expect "[data-cy=\"file-label-default.plugin-test.js\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-label-leto-modelizer.config.json\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"inactive-tab\"]" is "default.plugin-test.js"
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "leto-modelizer.config.json"
+    And  I wait 1 second
+
+    When I click on "[data-cy=\"file-tabs-container\"] [data-cy=\"file-label-default.plugin-test.js\"]"
+    Then I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "default.plugin-test.js"
+
+    When I set active file content to "[{ \"path\": \"default.plugin-test.js\", \"__class\": \"Component\", \"id\": \"image_9d1e0838\", \"name\": \"image_9d1e0838\", \"attributes\": [], \"type\": \"image\" }]"
+    And  I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Model"
+    And  I expect "[data-cy=\"modelizer-model-view-draw-root\"]" exists
+    And  I expect "[id^=\"image_\"]" exists
+    But  I expect "[id^=\"truck_\"]" not exists
+  
+  Scenario: Link two components (Model view) should update plugin file's content with new attributes properties (Text view)
+    When I click on "[data-cy=\"component-definition-paper\"]"
+    And  I click on "[data-cy=\"component-definition-image\"]"
+
+    When I click on "[id^=\"paper_\"]"
+    And  I click on "[id=\"create-link\"]"
+    And  I click on "[id^=\"image_\"]"
+    Then I expect ".link" exists
+
+    When I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Text"
+    And  I expect "[data-cy=\"modelizer-text-view\"]" exists
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"inactive-tab\"]" is "default.plugin-test.js"
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "leto-modelizer.config.json"
+    And  I wait 1 second
```

Comment:
and -> when
put it in the next paragraph

Reply:
done